.github/workflows/build.yaml

```diff
@@ -36,18 +36,33 @@ jobs:
         uses: actions/setup-go@v3
         with:
           go-version: ${{ matrix.go-version }}
+      - name: Setup Environment
```

Comment:
How does this help?

Reply:
GITHUB_ENV sets the environment for all subsequent sections.

So for all sections below, PATH, GOPATH and GOCACHE will be set.

that means that all 'go get' will use/utilize the cache.

Its the same cache that gets set in the makefile (and thus used by build of stacker-dynamic) and the same cache that is used by build.yaml.